src/firefly/js/ui/dynamic/EmbeddedPositionSearchPanel.jsx

```diff
@@ -155,32 +164,136 @@ export function EmbeddedPositionSearchPanel({
         ? <WrapperComponent {...additionalProps}>{internals}</WrapperComponent>
         : internals;
 
+    function SearchSummary({title, searchType, coords, radius, fileName, rows, showCoords, showRadius, showFileInfo, sx}) {
+        return (
+            <Stack sx={sx}>
+                {title && <><Typography color={'warning'} level='title'>{title}</Typography></>}
+                <Typography color={'neutral'} level='body-md'>
+                    <Typography component='span' color={'primary'}>Search Type</Typography>: {searchType}
+                    {showRadius && searchType === 'Cone' && <>
+                        , <Typography component='span' color={'primary'}>Search Radius</Typography>: {radius}
+                    </>}
```

Comment:
The combination of `<></>`, `, ` and nested Typography is hard to look at.  It would make it a lot easier to read if you define a simple function so you don't repeat yourself.
```js
  const keyVal = (k,v) => <>, <Typography component='span' color={'primary'}>{k}</Typography>: {v}</>;
```
I don't think `component='span'` is needed, but I guess you did it for a reason.